src/fragments/lib/storage/js/react-native-uploads.mdx

```diff
@@ -0,0 +1,15 @@
+Upload an image in your React Native app:
```

Comment:
```suggestion
## Media uploads

Upload an image in your React Native app:
```

Reply:
I merged both titles and have that available in the main uploads file now - please check my previous commit. I am closing out this suggestion.